determined-ai/determined
# fix: close button should not show in non-embedded log viewer [DET-7723]

## Description
fix: close button should not show in non-embedded log viewer [DET-7723]

The condition for showing the log view button currently is set to always be true. 



## Test Plan

1. Click "View Logs" from the embedded task view, there should be a "Close Logs" button displayed with an "X" visible in the screen.
2. View logs for a task from the Tasks page, no "Close Logs" icon should be visible.



## Commentary (optional)




## Checklist

See [Release Note](../docs/release-notes/README.md) for details.





[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DET-7723]: https://determinedai.atlassian.net/browse/DET-7723?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
